beast/tools/run/make_ast_inputs.py

```diff
@@ -26,9 +27,11 @@ def make_ast_inputs(beast_settings_info, flux_bin_method=True):
         if string: file name with beast settings
         if class: beast.tools.beast_settings.beast_settings instance
 
-    flux_bin_method : boolean (default=True)
-        If True, use the flux bin method to select SEDs.  If False, randomly
-        select SEDs from the model grid.
+    pick_method : string (default = "flux_bin_method")
+        If not specified, use the flux bin method to select SEDs as default.
+        If set to "random_seds", randomly select SEDs from the model grid.
+        If set to "suppl_seds", supplement input ASTs on top of the existing
+        input ASTs.
```

Comment:
add that the input ASTs are randomly selected from the list of non-selected models

Reply:
Done